tests/conftest.py

```diff
@@ -29,7 +29,7 @@ def enable_db_access_for_all_tests(db):
 
 
 @pytest.fixture
-def admin_user():
+def staff_user():
     return UserFactory(is_staff=True, is_superuser=True)
```

Comment:
This is probably my fault, but we don't need `is_superuser=True` here and it's misleading to have it, esp now that it's explicitly a staff user rather than an admin user.